localstack/services/sqs/provider.py

```diff
@@ -1276,28 +1276,28 @@ def _create_message_attribute_hash(message_attributes) -> Optional[str]:
     return hash.hexdigest()
 
 
-def get_queue_name_from_url(queue_url: str) -> str:
-    return queue_url.rstrip("/").split("/")[-1]
-
-
-def resolve_queue_name(
+def resolve_queue_location(
     context: RequestContext, queue_name: Optional[str] = None, queue_url: Optional[str] = None
-) -> str:
+) -> Tuple[str, Optional[str], str]:
```

Comment:
is the region still optional here?

Reply:
yep, when neither endpoint nor domain endpoint strategy is used